### ISSUE TITLE
Avoid duplicates in the "Uses"-list

### ISSUE THREAD

**Issue by bmaland** [hadley/crantastic#24]
E.g. [Rltools](http://crantastic.org/packages/RItools) both imports and suggests the xtable package

**Comment by bmaland** [comment]
Actually I'm not sure if this is a bug? Right now the page shows imports + depends in regular type, and suggested in italics. Maybe the UI for this part should be changed, since enhanced packages also should be displayed.

**Comment by hadley** [comment]
Good point.  I don't think it's worth breaking up imports and depends, but suggests should be different, and maybe enhances should be in its own list.
